addon/components/uni-calendar.js

```diff
@@ -0,0 +1,8 @@
+import Component from '@ember/component';
+import layout from '../templates/components/uni-calendar';
+
+export default Component.extend({
+  layout,
+
+  showAbove: false
```

Comment:
I would suggest `position: 'above' | 'below'` to be more flexible in the future  :)